tool/tsh/tsh.go

```diff
@@ -370,6 +373,15 @@ func Run(args []string, opts ...cliOption) error {
 	appConfig.Flag("format", fmt.Sprintf("Optional print format, one of: %q to print app address, %q to print CA cert path, %q to print cert path, %q print key path, %q to print example curl command.",
 		appFormatURI, appFormatCA, appFormatCert, appFormatKey, appFormatCURL)).StringVar(&cf.Format)
 
+	// proxy.
+	proxy := app.Command("proxy", "proxy")
+	proxySSH := proxy.Command("ssh", "proxy ssh")
+	proxySSH.Arg("[user@]host", "Remote hostname and the login to use").Required().StringVar(&cf.UserHost)
+	proxySSH.Flag("cluster", clusterHelp).StringVar(&cf.SiteName)
+	proxyDB := proxy.Command("db", "proxy db")
```

Comment:
```suggestion
	proxyDB := proxy.Command("db", "Start local TLS proxy for database connections when using Teleport in single-port mode")
```

or something like this.